ocaml/database/db_cache.ml

```diff
@@ -27,7 +27,7 @@ module Remote_db : DB_ACCESS = Db_rpc_client_v1.Make(struct
     let initialise () =
```

Comment:
Or could we just pass the uri to the `initialise` function and then onward to `execute_remote_fn`?